src/mcts/node.cc

```diff
@@ -358,6 +426,8 @@ void NodeTree::MakeMove(Move move) {
   current_head_->ReleaseChildrenExceptOne(new_head);
   current_head_ =
       new_head ? new_head : current_head_->CreateSingleChildNode(move);
+  if (current_head_->GetParent()) current_head_->GetOwnEdge()->ClearEdge();
```

Comment:
Why do we need that? Why would something need to look somewhere above search head?

Reply:
Sometimes we can prove bounds for root. Lets say root is lowerbounded -> we know that we have at least a draw, but might have something better. In these cases we do not need to search moves from root that are upperbounded (at maximum a draw). As we store all information about bounds/certainty in the edge leading to the node, I fetch it there. The only time you do not have that info is in the startposition of chess and if you provide a fen. But in gameplay you usually do. Also if we proved that a later move has bounds, and we finally arrive at that position it would be a pity to loose that info.